Modules/_testcapimodule.c

```diff
@@ -2143,10 +2199,20 @@ static PyObject *
 test_null_strings(PyObject *self, PyObject *Py_UNUSED(ignored))
 {
     PyObject *o1 = PyObject_Str(NULL), *o2 = PyObject_Str(NULL);
```

Comment:
In Python 2 this test tested `PyObject_Str` and `PyObject_Unicode`. In Python 3 there is no `PyObject_Unicode` and there is no need to call `PyObject_Str` twice. Just return `PyObject_Str(NULL)`.